tests/std/tests/GH_004618_mixed_operator_usage_keeps_statistical_properties/test.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cmath>
+#include <cstddef>
+#include <numeric>
+#include <random>
+#include <vector>
+
+
+static const size_t N = 1000000;
+
+void check_results(std::vector<double> const& generated, double expected_mean, double expected_variance) {

```

Comment:
`expected_mean` and `expected_variance` could be `const`. We don't always rigorously follow this practice (especially in test code), but it's preferable especially in longer functions because it prevents mistakes and draws attention to parameters that actually are modified.